.travis/build

```diff
@@ -8,10 +8,9 @@ export DEBIAN_FRONTEND=noninteractive
 
 # Dependencies.
 echo "deb [trusted=yes] http://packages.osrfoundation.org/gazebo/ubuntu-stable bionic main" > /etc/apt/sources.list.d/gazebo-stable.list
-echo "deb [trusted=yes] http://packages.osrfoundation.org/gazebo/ubuntu-prerelease bionic main" > /etc/apt/sources.list.d/gazebo-prerelease.list
 echo "deb [trusted=yes] http://packages.ros.org/ros/ubuntu bionic main" > /etc/apt/sources.list.d/ros-latest.list
 apt-get update -qq
-apt-get install -qq -y libignition-common3-dev \
+apt-get install -qq -y libignition-msgs3-dev \
```

Comment:
@scpeters , I noticed that the code was relying on the msgs version pulled by transport. Do you think it would be better to add explicit calls for `find_package(ignition-msgs3 QUIET REQUIRED)` since we will make the installation explicit too?

Reply:
yeah, that would be good, since it is being used directly

Reply:
Done 78ceea90035d604a58ea76e4ca0250a552a33bf1

Reply:
change dependency in the -release repo 2c87ee0